portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,31 +36,33 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<String> greetings = new ArrayList<String>();
-    String json = convertToJsonUsingGson(greetings);
+    Query query = new Query("comment");
+    
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    ArrayList messages = new ArrayList<>();
+    for (Entity entity : results.asIterable()){
+        long id = entity.getKey().getId();
+        String msg = (String) entity.getProperty("comment");
+        messages.add(msg);
+    }
+    String json = convertToJsonUsingGson(messages);
+
     response.setContentType("text/html;");
     response.getWriter().println(json);
   }
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get the input from the form.
-    String text = request.getParameter("text-input");
-    long timestamp = System.currentTimeMillis();
-
-    // Break the text into individual words.
-    String[] words = text.split("\\W+");
-
-    // Respond with the result.
-    response.setContentType("text/html;");
-    response.getWriter().println(Arrays.toString(words));
+    String text = request.getParameter("comment");
```

Comment:
nit: I think you can be a little more descriptive with some variable names, like 'text' in this case. You could use something like requestComment or requestText or comment. 

This can be important when someone else works on your code, or when you come back to your code after a while. A descriptive name can reduce confusion between variables and make your code more readable.